packages/api/taskOrderFiles/createTaskOrderFile.ts

```diff
@@ -0,0 +1,64 @@
+import { APIGatewayProxyEvent, APIGatewayProxyResult } from "aws-lambda";
+import { v4 as uuid } from "uuid";
+import { ApiSuccessResponse, ErrorResponse, ErrorStatusCode, SuccessStatusCode } from "../utils/response";
+import { PutObjectCommand, S3Client } from "@aws-sdk/client-s3";
+import { FileMetadata, FileScanStatus } from "../models/FileMetadata";
+import * as parser from "lambda-multipart-parser";
+import { ErrorCodes } from "../models/Error";
+
+const bucketName = process.env.PENDING_BUCKET;
+
+/**
+ * Creates a new Task Order File
+ *
+ * @param event - The POST request from API Gateway
+ */
+export const handler = async (event: APIGatewayProxyEvent): Promise<APIGatewayProxyResult> => {
+  const result = await parser.parse(event);
+  if (result.files.length !== 1) {
+    return new ErrorResponse(
+      { code: ErrorCodes.INVALID_INPUT, message: "Expected exactly one file." },
+      ErrorStatusCode.BAD_REQUEST
+    );
+  } else {
```

Comment:
```
if (condition) {
  return;
}
else {
  // stuff
}
```

Can be cleaned up to:

```
if (condition) {
  return;
}
// stuff
```

There are a few places to make this improvement here.

Reply:
Ah good call. Fixed that as well. Ultimately something like 

```
validate(event);  // this will throw errors
return await doTheThing(event);
```

would be nice to make it more readable and testable. But as mentioned I'll avoid any further optimizations in validation for now.